src/hct_mis_api/apps/payment/celery_tasks.py

```diff
@@ -338,6 +338,36 @@ def payment_plan_apply_engine_rule(self: Any, payment_plan_id: str, engine_rule_
         raise self.retry(exc=e)
 
 
+@app.task(bind=True, default_retry_delay=60, max_retries=3)
+@log_start_and_end
+@sentry_tags
+def update_exchange_rate_on_release_payments(self: Any, payment_plan_id: str) -> None:
+    from hct_mis_api.apps.payment.models import Payment, PaymentPlan
+
+    payment_plan = get_object_or_404(PaymentPlan, id=payment_plan_id)
+    set_sentry_business_area_tag(payment_plan.business_area.name)
+    try:
+        payment_plan.exchange_rate = payment_plan.get_exchange_rate()
```

Comment:
Please review all other occurrences of pp.get_exchange_rate()
We should probably use pp.exchange_rate in all of these places (except pp.open() ?)